test/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM quay.io/podman/stable
+
+WORKDIR /project
+
+# Setup Yggdrasil & device-worker repo:
+RUN curl -k https://copr.fedorainfracloud.org/coprs/ondramachacek/yggdrasil/repo/fedora-34/ondramachacek-yggdrasil-fedora-34.repo -o /etc/yum.repos.d/yggd.repo
```

Comment:
We should have a non-personal copr account - not necessarily before this PR is merged.